torchvision/transforms/functional.py

```diff
@@ -842,7 +842,7 @@ def adjust_hue(img: Tensor, hue_factor: float) -> Tensor:
 
     Args:
         img (PIL Image or Tensor): Image to be adjusted.
-            If img is torch Tensor, it is expected to be in [..., 3, H, W] format,
+            If img is torch Tensor, it is expected to be in [..., 1 or 3, H, W] format,
```

Comment:
> If img is PIL Image mode "1", "L", "I", "F" and modes with transparency (alpha channel) are not supported.

Same here, "L" is not now support

Reply:
On the pillow side, the `mode="L"` is supported before this PR as you can see in
https://github.com/pytorch/vision/blob/011eedfbf6a86d22da443141139519493e33eec5/torchvision/transforms/functional_pil.py#L73-L79

However, I've tested with pillow >= 8. I don't know about previous version though.

Reply:
Well, my request here is to change the sentence below which says that PIL Image model "L" is not supported.

Reply:
@VinhLoiIT please address these unresolved conversations

Reply:
Yes, I've updated the docstring of the ColorJitter class

Reply:
This one is for `adjust_hue`